draft-ietf-httpbis-messaging-latest.xml

```diff
@@ -249,14 +249,15 @@
 <iref item="headers"/>
 <iref item="header field"/>
 <t>
-   All HTTP/1.1 messages consist of a start-line followed by a sequence of
+   All HTTP/1.1 messages consist of a start-line followed by a CRLF and a
```

Comment:
I think this should be:

> All HTTP/1.1 messages consist of an initial line of control data (terminated by CRLF) followed by optional header fields, an empty line terminating the header section, and an optional message body.

Alternatively, go ahead and merge this PR and we can make an editorial change later.

Reply:
Yes, I'd prefer to do that separately, as it looses the RFC 5322 reference...